.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
   - 6
-script: yarn && yarn run lint && yarn test
+script: npm run lint && npm test
```

Comment:
Can this also `npm run build` -- I want to guard against any possible issues creating a build